release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2068,7 +2068,7 @@ In the following tables, features are marked with the following statuses:
 
 * A regression in the behaviour of `libreswan` caused some nodes with IPsec enabled to lose communication with pods on other nodes in the same cluster. To resolve this issue, consider disabling IPsec for your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-43713[*OCPBUGS-43713*])
 
-* There is a known issue with {op-base-system} 8 worker nodes that  use `cgroupv1` Linux Control Groups (cgroup). The following is an example of the error message displayed for impacted nodes: `UDN are not supported on the node ip-10-0-51-120.us-east-2.compute.internal as it uses cgroup v1.` As a workaround, users should migrate worker nodes from `cgroupv1` to `cgroupv2`. (link:https://issues.redhat.com/browse/OCPBUGS-49933[*OCPBUGS-49933*])
+* There is a known issue with {op-system-base} 8 worker nodes that  use `cgroupv1` Linux Control Groups (cgroup). The following is an example of the error message displayed for impacted nodes: `UDN are not supported on the node ip-10-0-51-120.us-east-2.compute.internal as it uses cgroup v1.` As a workaround, users must reconfigure their nodes from `cgroupv1` to `cgroupv2` before creating a user-defined network. For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#nodes-cluster-cgroups-2[Configuring Linux cgroup]. (link:https://issues.redhat.com/browse/OCPBUGS-49933[*OCPBUGS-49933*])
```

Comment:
Same comment as 88906. We should not limit the surface area of this bug to only rhel8 nodes. this applies to any node using cgroupsv1. cc @npinaeva.

Reply:
@asood-rh here is the message you asked about in the other PR #88906